src/core/packages/saved-objects/import-export-server-internal/src/import/import_saved_objects.ts

```diff
@@ -95,6 +100,8 @@ export async function importSavedObjectsFromStream({
     objectLimit,
     supportedTypes,
     managed,
+    typeRegistry,
+    createAccessControlImportTransforms,
```

Comment:
Handling access control in the importStateMap means that the ownership will already be decided by the time the objects are passed to `createSavedObjects`.

> any new objects (that support access control) created as a result of import will be owned by the importing user and will be in default mode

`createNewCopies: true; overwrite: false`

> Any objects being overwritten as a result of import will maintain existing ownership and mode

`createNewCopies: false; overwrite: true`

However, `createAccessControlImportTransforms` doesn't take `overwrite` or `createNewCopies` into account, so how do we know what choice is being made?